backend/curation/api/curation-api.yml

```diff
@@ -974,6 +974,7 @@ components:
           $ref: "#/components/schemas/donor_id"
         explorer_url:
           $ref: "#/components/schemas/explorer_url"
+          required: true
```

Comment:
shouldn't we be using this feature? 🤔 cc @Bento007

Reply:
I found it best to only use `required` when sanitizing requests from the user. Using it on our responses force us to comply with the format, makes our code brittle, and results more 500 responses for our users. It's better to return a response that is missing some fields than throw a 500 at our users. Using the openapi to enforce our output via "required" is heavy handed and ultimately results in a worse experience for our users, because they either get all the info or none of the info based on a mistake outside of their control. We should be validating our responses in unit tests anyways, so using "required" for outputs _should_ not be required. 

If we were not using connexions which interprets and enforces the openapi spec I would say use required since the spec would be more for documentation purposes.

Reply:
Ok, I buy that. Cool.